mods/seasonal/statuses.js

```diff
@@ -19,6 +19,14 @@ exports.BattleStatuses = {
 			this.add('c|&kamikaze|NANI!');
 		},
 	},
+	panpawn: {
+		exists: true,
+		onStart: function () {
+			this.add('c|%panpawn|hello darkness my old friend,,,');
+		}, //lack on switch out phrase
```

Comment:
```javascript
onSwitchOut: function () {
    this.add('c|%panpawn|... >:^(');
}
```
(dont c/p this as it will fail travis by using spaces instead of tabs)